### ISSUE TITLE
C#7.2: Missing "in" parameter and readonly

### ISSUE THREAD

**Issue by KristianWedberg** [dotnet/docfx#3125]
**DocFX Version Used**: 2.37.1

**Template used**: `default`

**Steps to Reproduce**:

```C#
    public class InOut
    {
        public readonly struct ROS
        {
            public int Value { get; }
        }

        public void TestInOut(in ROS myIn, out int myOut)
        {
            myOut = 42;
        }
    }
```

**Expected Behavior**:

- `readonly` is shown in the Syntax section of the `ROS` struct
- `in` is shown on the parameter in the _header_ and Declaration section of the `TestInOut()` method (especially since `out` _is_ shown)

**Actual Behavior**:

- `readonly` is _not_ shown in the Syntax section of the `ROS` struct:

![image](https://user-images.githubusercontent.com/11193804/43370850-94baeb32-937e-11e8-8d72-46c7baf4faf4.png)

- `in` is _not_ shown on the parameter in the _header_ section of the `TestInOut()` method (but _is_ shown in the Declaration section):

![image](https://user-images.githubusercontent.com/11193804/43370839-5b124dee-937e-11e8-939e-1d2e67768d96.png)

**Comment by herohua** [comment]
@TianqiZhang will you be able to help on this one in near future? Thanks!

**Comment by KristianWedberg** [comment]
Any chance of getting `readonly` and `in` resolved?

**Comment by yufeih** [comment]
Fixed by #6805
